provider/testdata/data/json/props/exemplarchars/main@1/ja.json

```diff
@@ -0,0 +1,3514 @@
+      38750,
+      38751,
+      38754,
+      38755,
+      38761,
+      38762,
+      38772,
+      38773,
+      38867,
+      38868,
+      38899,
+      38900,
+      38907,
+      38908,
+      38911,
+      38912,
+      38914,
+      38916,
+      38917,
+      38919,
+      38920,
+      38921,
+      38936,
+      38937,
+      38956,
+      38958,
+      38971,
+      38973,
+      38996,
+      38998,
+      39000,
+      39001,
+      39006,
+      39007,
+      39015,
+      39016,
+      39080,
+      39081,
+      39131,
+      39132,
+      39135,
+      39136,
+      39138,
+      39139,
+      39151,
+      39152,
+      39154,
+      39155,
+      39173,
+      39174,
+      39178,
+      39179,
+      39180,
+      39181,
+      39187,
+      39188,
+      39208,
+      39209,
+      39318,
+      39319,
+      39321,
+      39322,
+      39340,
+      39341,
+      39376,
+      39377,
+      39378,
+      39379,
+      39438,
+      39439,
+      39442,
+      39444,
+      39472,
+      39473,
+      39514,
+      39515,
+      39592,
+      39593,
+      39608,
+      39609,
+      39620,
+      39621,
+      39640,
+      39641,
+      39658,
+      39659,
+      39729,
+      39730,
+      39740,
+      39741,
+      39746,
+      39747,
+      39749,
+      39750,
+      39764,
+      39765,
+      39770,
+      39771,
+      39854,
+      39855,
+      39912,
+      39913,
+      40165,
+      40166,
+      40180,
+      40181,
+      40335,
+      40336,
+      40372,
+      40373,
+      40575,
+      40576,
+      40595,
+      40596,
+      40599,
+      40600,
+      40614,
+      40615,
+      40634,
+      40636,
+      40644,
+      40645,
+      40658,
+      40659,
+      40665,
+      40666,
+      40723,
+      40724,
+      40763,
+      40764,
+      40802,
+      40803
+    ],
+    "str_list": [
+      "ゝヽ",
+      "ゞヾ",
+      "ぁァ",
+      "あア",
+      "ぃィ",
+      "いイ",
+      "ぅゥ",
+      "うウ",
+      "ぇェ",
+      "えエ",
+      "ぉォ",
+      "おオ",
+      "かカ",
+      "がガ",
+      "きキ",
+      "ぎギ",
+      "くク",
+      "ぐグ",
+      "けケ",
+      "げゲ",
+      "こコ",
+      "ごゴ",
+      "さサ",
+      "ざザ",
+      "しシ",
+      "じジ",
+      "すス",
+      "ずズ",
+      "せセ",
+      "ぜゼ",
+      "そソ",
+      "ぞゾ",
+      "たタ",
+      "だダ",
+      "ちチ",
+      "ぢヂ",
+      "っッ",
+      "つツ",
+      "づヅ",
+      "てテ",
+      "でデ",
+      "とト",
+      "どド",
+      "なナ",
+      "にニ",
+      "ぬヌ",
+      "ねネ",
+      "のノ",
+      "はハ",
+      "ばバ",
+      "ぱパ",
+      "ひヒ",
+      "びビ",
+      "ぴピ",
+      "ふフ",
+      "ぶブ",
+      "ぷプ",
+      "へヘ",
+      "べベ",
+      "ぺペ",
+      "ほホ",
+      "ぼボ",
+      "ぽポ",
+      "まマ",
+      "みミ",
+      "むム",
+      "めメ",
+      "もモ",
+      "ゃャ",
+      "やヤ",
+      "ゅュ",
+      "ゆユ",
+      "ょョ",
+      "よヨ",
+      "らラ",
+      "りリ",
+      "るル",
+      "れレ",
+      "ろロ",
+      "ゎヮ",
+      "わワ",
+      "ゐヰ",
+      "ゑヱ",
+      "をヲ",
+      "んン",
+      "万-下",
```

Comment:
@echeran Can you take a look at the Japanese data? Looks like it might have some range operators. In general the Japanese data is about 100x bigger than most other data, which I guess  makes sense, but we should check to make sure.

Reply:
+1.  I also forgot to parse the exemplar character syntax that allows `{...}` syntax to wrap literals, whether for single characters or for strings (character sequences).  Ex: `[a b c č ć d {dž} đ e ...]` needs to parse and return `dž`, not `{dž}`, as an element.